client/game/Timer.jsx

```diff
@@ -1,9 +1,23 @@
 import React from "react";
 
 import { StageTimeWrapper } from "meteor/empirica:core";
+
 import Timer from "./Timer.jsx";
 
 class timer extends React.Component {
+  startTimer = () => {
+    const { remainingSeconds } = this.props;
+    const timer = remainingSeconds;
+    let minutes;
+    let seconds;
+    minutes = parseInt(timer / 60, 10);
+    seconds = parseInt(timer % 60, 10);
+
+    minutes = minutes < 10 ? "0" + minutes : minutes;
+    seconds = seconds < 10 ? "0" + seconds : seconds;
```

Comment:
You can use padStart for this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart